packages/webviz-core/package.json

```diff
@@ -10,10 +10,10 @@
     "@simonwep/pickr": "1.4.7",
     "@trust/webcrypto": "0.9.2",
     "antd": "3.26.5",
-    "chart.js": "2.7.3",
+    "chart.js": "davidswinegar/Chart.js#ab170d249db7cae3313d9e2515ef35283c9bb5e8",
```

Comment:
Are we going to use this version in the future or is there a plan to make a PR to the official package?

Reply:
There is a plan to make a PR to the official package later